pyproject.toml

```diff
@@ -61,6 +62,9 @@ dev-dependencies = [
     "mypy>=1.0.0",
 ]
 
+[tool.uv.sources]
+refraim = { git = "https://github.com/fraim-dev/refraim.git" }
+
```

Comment:
This was originally added to a benchmark dependency group, however, this breaks development for people that don't have access to refraim so I removed the `benchmark` group for now.

I may need to follow up on a fix for this.